### ISSUE TITLE
`closeThreshold` not working as expected when `direction="left"`

### ISSUE THREAD

**Issue by yodhcn** [emilkowalski/vaul#385]
When swiping from right to left, `swipeAmount` is negative.
https://github.com/emilkowalski/vaul/blob/40d3a69825df25e04ffe1e52c99c3461d785f31a/src/index.tsx#L578

Using absolute values ​​may solve this problem.
```
if (Math.abs(swipeAmount) >= visibleDrawerHeight * closeThreshold) {
```

**Comment by yodhcn** [comment]
In addition, when the `direction` is horizontal, `visibleDrawerWidth` needs to be calculated instead of `visibleDrawerHeight`.

**Comment by emilkowalski** [comment]
Merged #386
